packages/create-atomic/index.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
```

Comment:
Why do you need the shebang? 🤔

Reply:
just copied from their docs too

Reply:
I'm curious abt it, because shebang does not work on windows soo...

Reply:
Seems like a good thing to keep https://stackoverflow.com/a/33510581

This way, maybe when i install `npm i -g @coveo/create-atomic` and then run `create-atomic myproject` it will invoke node correctly 🤷  I haven't tested, just thinking out loud